.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: scala
+
+scala:
+
+   - 2.10.0
+
+script: sbt test
```

Comment:
I would suggest rebasing, squashing commits and force pushing to this feature branch. @danhanson Can you help with this? There are some other files to rebase as well, like the auto-gen'd test files.